documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -14,13 +14,46 @@ You must upgrade to the next release without skipping a release, for example, fr
 
 .Procedure
 
-. Upgrade the {operator-name}.
+. Change the update channel to *release-v2.2.0*.
 +
 ifeval::["{build}" == "upstream"]
-See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://docs.okd.io/latest/operators/admin/olm-upgrading-operators.html#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
 endif::[]
 ifeval::["{build}" == "downstream"]
-See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-upgrading-operators[Upgrading installed Operators] in the {ocp} documentation.
+See link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html-single/operators/index#olm-changing-update-channel_olm-upgrading-operators[Changing update channel] in the {ocp} documentation.
+endif::[]
+
+. Restart the `CatalogSource` pod:
+
+.. In the the {ocp-short} web console, click *Operators* -> *Installed Operators*. Click *Forklift Operator* to view the *Operator details* page.
+
+..  On the *Subscriptions* tab, note the catalog source, for example, `redhat-operators`.
+..  From the command line, retrieve the catalog source pod:
++
+[source,terminal,subs=attributes+]
+----
+$ {oc} get pod -n openshift-marketplace | grep <catalog_source>
+----
++
+The retrieved pod name includes the *CatalogSource* name on the *Subscriptions* tab in the the {ocp-short} web console. For example, if the *CatalogSource* name is `redhat-operators`, its pod might be `redhat-operators-mgrrm`.
```

Comment:
Remove this sentence because it will be in the callout.

Reply:
@apinnick  But where do I give an example of the pod name? It won't be in the callout.

Reply:
I don't think it's necessary. The user can figure it out.